### ISSUE TITLE
Error after installing verison 5.0.4. Axios related.

### ISSUE THREAD

**Issue by bretmorris** [contentful/contentful.js#203]
```
ERROR in ./node_modules/contentful/dist/es-modules/contentful.js
Module not found: Error: Can't resolve 'axios' in 'my_path/node_modules/contentful/dist/es-modules'
```

**Comment by Khaledgarbaya** [comment]
it is now [fixed](https://github.com/contentful/contentful.js/pull/204) I am terribly sorry about this :(

**Comment by bradley-marker-ck** [comment]
I still have issues on 5.0.5. Is this fix supposed to be in 5.0.5?

**Comment by bradley-marker-ck** [comment]
On second look, my issue is different. Rather than a resolution error, the proxy behavior seems broken again, as if the fix in the contentful branch were missing.
